pkg/api/types.go

```diff
@@ -1205,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
```

Comment:
The thought I had was to have an admission controller add the label to PVs based off of the NodeName field, if it wasn't already set.  There's been some push back against having the system validate specific labels, because theoretically any label could be used, not just the hostname label.

I also want the NodeName to be a required field in the volume spec so I can ensure that the volume is only ever mounted on the correct node, in case some user or system error happens and the volume gets scheduled to the wrong node.